main.go

```diff
@@ -74,8 +70,21 @@ func defaultWorker() {
 	sendKey(serverEndpoint, workerInstance.Id)
 
 	for {
-		if !isTokenValid(workerInstance.Token) {
+		err := utils.CheckToken(workerInstance.Token)
```

Comment:
shouldn't CheckToken return a bool?

Reply:
should we rename to isValid(Token)?

Reply:
For both questions, I think this way seems more like golang way to do things, don't you?

Reply:
maybe if so `token.Check` it would be more idiomatic, no?

Reply:
@emanueljoivo we would need to create an abstraction for the token, is it you're suggesting?

Reply:
my point is to return a bool instead of another type. you guys @raonismaneoto and @emanueljoivo can decide the more idiomatic way to do so